src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -33,6 +34,44 @@ using namespace mlpack::ann;
 
 BOOST_AUTO_TEST_SUITE(LossFunctionsTest);
 
+BOOST_AUTO_TEST_CASE(PoissonNegativeLogLikelihoodTest)
+{
+  arma::mat log_input, output, target;
+  PoissonNegativeLogLikelihoodLoss<> module;
+
+  // Test the Forward function on a user generated input and compare it against
+  // the manually calculated result.
+  log_input = arma::mat("1.0 1.0 3.0 2.0 -1.0 1.0 -1.0 0.0");
```

Comment:
Hi Could you rename this variable as input as below you have used input but not declared it.

Reply:
Refer line 50,56,57.